src/player.py

```diff
@@ -31,6 +33,11 @@ def update(self, delta_time: float):
             self.sprite.center_y += delta_time * self.drive_speed
         if self.input.y_axis.value < 0:
             self.sprite.center_y -= delta_time * self.drive_speed
+        if self.input.accelerate_button.value:
+            self.player_health -= 1
+        if self.input.brake_button.value:
+            self.player_health = 100
```

Comment:
I don't think this is supposed to be committed?  Or was the intent to merge it for now, and then remove it when MikeD's driving code is added to the game?